config/settings.yml

```diff
@@ -380,6 +380,11 @@ evss:
   url: https://csraciapp6.evss.srarad.com
   service_name: "wss-form526-services-web"
   alternate_service_name: "wss-form526-services-web-v2"
+  dvp:
+    current_env: qa
+    qa:
+      url: https://blue.qa.lighthouse.va.gov
+      service_name: /wss-form526-services-web
```

Comment:
if you want this specific to an environment without having to manually do so, we could add it [here](https://github.com/department-of-veterans-affairs/devops/blob/02d356eae2b2a5627ada6b946ae1f3b18679a60e/ansible/deployment/config/vets-api/dev-settings.local.yml.j2#L46).

Reply:
That makes sense - I'll remove this here and add it to the `devops` project.

Reply:
The only issue I see with this is I don't think there's a way to default a flipper feature to false, so we'll need to define it for every environment, the problem being that we only have a QA instance currently. I may need to add something to `flipper.rb` to allow setting a default value so that this doesn't break prod or the other environments.

Reply:
@FonzMP I made the change you suggested here https://github.com/department-of-veterans-affairs/devops/pull/12309